GameServer/Logic/GameObjects/Unit.cs

```diff
@@ -53,6 +55,8 @@ public class Unit : GameObject
         internal const long UPDATE_TIME = 500;
 
         protected Stats stats;
+        protected InventoryManager Inventory { get; set; }
```

Comment:
the whole InventoryManager is protected, not sure if that would make any difference